pkg/porter/archive.go

```diff
@@ -336,6 +336,9 @@ func (ex *exporter) addImage(base bundle.BaseImage) error {
 // If the name contains a path separator, all path separators will be replaced with "-".
 func (ex *exporter) createArchiveFolder(name string) (string, error) {
 	cleanedPath := strings.ReplaceAll(afero.UnicodeSanitize(name), string(os.PathSeparator), "-")
+	cleanedPath = strings.ReplaceAll(cleanedPath, "/", "-") // this is needed because on windows, if the bundle name is from a reference
+	// e.g. "ghcr.io/getporter/examples/porter-hello:v0.2.0", the name will be "examples/porter-hello",
+	// so we still need to replace the / with -, which the above line doesn't do (on windows)
```

Comment:
I'm wondering why this respected the OS's PathSeparator to begin with...?

Reply:
I think we can just use '/', no need for checking PathSeparator here. https://github.com/getporter/porter/pull/2154/files